Cannot run performance test via "npm run"

This is not a duplicate of #229 

Output:

```
$ npm run benchmark

> eventemitter2@5.0.1 benchmark /Users/ronkorving/Projects/EventEmitter2
> node --reporter test/perf/benchmark.js

node: bad option: --reporter
npm ERR! code ELIFECYCLE
npm ERR! errno 9
npm ERR! eventemitter2@5.0.1 benchmark: `node --reporter test/perf/benchmark.js`
npm ERR! Exit status 9
npm ERR! 
npm ERR! Failed at the eventemitter2@5.0.1 benchmark script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ronkorving/.npm/_logs/2018-01-31T01_45_38_512Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
   "scripts": {
     "test": "nodeunit test/simple/ test/wildcardEvents/",
     "coverage": "nyc --check-coverage npm run test",
-    "benchmark": "node --reporter test/perf/benchmark.js"
+    "benchmark": "node test/perf/benchmark.js"
   },
   "files": [
     "lib/eventemitter2.js",

